--- parser.y	2020-07-07 15:39:39.403691645 +0200
+++ /media/vbox/Users/sjanssen/Git/jlab/gapc/src/parser.y	2020-07-07 15:01:57.000000000 +0200
@@ -68,15 +68,15 @@
 
 %skeleton "lalr1.cc"
 %defines  /* yacc -d */
-%define api.parser.class {Parser}
-%define api.location.type { Loc }
+%define "parser_class_name" "Parser"
+%define "location_type" "Loc"
 %parse-param { Driver& driver }
 %parse-param { yy::Parser::token_type start_symbol }
 %lex-param   { yy::Parser::token_type start_symbol }
 %locations
 %debug    /* yacc -t */ /* FIXME */
 %verbose  /* yacc -v */
-%define parse.error verbose
+%error-verbose 
 
 %initial-action
 {
